.gitignore

```diff
@@ -5,4 +5,4 @@
 
 # misc
 .DS_Store
-
+.vscode
```

Comment:
Should be a global gitignore has not directly related to this project

Reply:
What's the difference between `.vscode` and `.DS_Store` in this case ?

Reply:
non go global gitignore

Reply:
Alright. Should I remove the entire file though?

Reply:
Done.